shared/components/NavigationBar/index.scss

```diff
@@ -1,57 +1,150 @@
-.NavigationBar {
-  background: $grey;
-
-  li {
-    list-style-type: none;
+body {
+  background-color: $conference-color;
+}
 
-    @media (min-width: 660px) {
-      width: 280px;
+.NavigationBar {
+  background-color: $grey;
+  display: flex;
+  &__links-container {
+    width: 100%;
+    ul {
+      display: none; /* Not displayed on mobile unless the hamburger menu is opened */
+      list-style: none;
+      margin: 0;
+      padding: 0;
+      flex-direction: column;
+      li {
+        a {
+          padding: 14px 0px 14px 80px;
+          display: block;
+          font-weight: bold;
+          font-size: 18px;
+          color: $conference-color;
+          text-decoration: none;
+          &:hover {
+            background: $conference-color;
+            color: $grey;
+          }
+        }
+        a.active {
```

Comment:
Try to avoid unneeded element qualifiers, it bumps up specitivity, making it hard to extend this CSS later